stripe/_stripe_object.py

```diff
@@ -383,6 +383,9 @@ def _refresh_from(
     def api_base(cls) -> Optional[str]:
         return None
 
+    @_util.deprecated(
```

Comment:
Aren't we already removing a bunch of other stuff? Or do you think this one is too popular?

Reply:
Also, what happens when someone asks for an alternative?

Reply:
This method seemed more likely to be used IMO. But we did remove something similar in stripe-java (https://github.com/stripe/stripe-java/blob/6c3dc053cb7945c8a8fa20ffb828b0a4cd7225fa/CHANGELOG.md?plain=1#L255)

Reply:
Hmm, good point. I guess the alternative is `APIRequestor.request(...)` with the StripeObject's request options. But that goes to show that this really doesn't make sense to use. I'll remove it like we did for stripe-java